API/ChatgptService/__init__.py

```diff
@@ -18,50 +18,67 @@
+
+
 def main(req: func.HttpRequest) -> func.HttpResponse:
     if req.method != "POST":
         return func.HttpResponse("Chatgpt Service only supports PUT request.", status_code=200)
     if req.method == "POST":
         logging.info(
             'Chatgpt Service HTTP trigger function processed a request.')
-        response_data = {"content": None, "history_msg_list": None}
         # the param dict of the chatgpt service
         user_msg = get_param_str(req, "user_msg", required=True)
         history_msg = get_param_list(req, "history_msg")
-        chatgpt_service_params = initialize_chatgpt_service_params()
-        all_user_msg = []
-        if history_msg:
-            chatgpt_service_params["messages"].extend(history_msg)
-            all_user_msg = [msg["content"]
-                            for msg in history_msg if msg["role"] == "user"]
-        all_user_msg.append(user_msg)
-        chatgpt_service_params["messages"].append(
-            {"role": "user", "content": "\n".join(all_user_msg)})
-        try:
-            response = openai.ChatCompletion.create(**chatgpt_service_params)
-            content = response["choices"][0]["message"]["content"]
-            history_msg.append({"role": "user", "content": user_msg})
-            history_msg.append({"role": "assistant", "content": content})
-            response_data["content"] = json.loads(content.replace("'", '"'))
-            response_data["history_msg"] = history_msg
-            return func.HttpResponse(generate_response(data=response_data, status=200))
-        except:
+        openai_prompt = get_param_str(req, "openai_prompt")
+        openai_prompt = get_default_prompt(openai_prompt) if not openai_prompt else json.loads(openai_prompt)
```

Comment:
`openai_prompt` is the Prompt used to extract keywords from the question, right? Is this design for the convenience of testing Prompt?